src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -38,6 +38,22 @@ public Datastore() {
     datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
+  /** Stores the Message in Datastore. */
+  public void storeMessage(Message message) {
```

Comment:
This looks like an out-of-date version of the `storeMessage()` function on line 91 below. This one is missing the `parent` property. I think you can probably just delete this version of the funciton.